gui_qt/settings_schema.py

```
@@ -219,7 +219,7 @@ def recommended_value(self) -> SettingValue:
         "batch_temperature",
         ("batch", "temperature"),
         "批量 temperature",
-        "Batch 请求的温度；越低越稳定。",
+        "仅供仍支持采样参数的模型；新 Gemini 型号可能忽略或拒绝该值，数值更低不保证更稳定。",
```

Ruff (0.16.1)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)